frontend/openapi-ts.config.ts

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from "@hey-api/openapi-ts"
+
+export default defineConfig({
+  client: "legacy/axios",
+  input: "./openapi.json",
+  output: "./src/client",
+  // exportSchemas: true,
+  plugins: [
+    {
+      name: "@hey-api/sdk",
+      // NOTE: this doesn't allow tree-shaking
```

Comment:
I've left this as a note, maybe we can change this setting in future (though it would require us to change quite a bit of code 😊)